Lib/Rabbit4000/tcpip/DMAETH100.LIB

```diff
@@ -1584,25 +1653,20 @@ int dmaeth100_receive(_DMAEth100Config * nic)
       call dmaeth100_alloc_rx
       jr    c,.cont1
 
+		/* Don't have to do this with the R6000...tx fire will do it */
 #if _RAB6K
-		ld		hl,(ix+[_decb]+rxbit)
-		ex		de,hl
-		ioi	ld	hl,(DMCSLR)
-		and	hl,de
+		;ld		hl,(ix+[_decb]+rxbit)
+		;ex		de,hl
+		;ioi	ld	hl,(DMCSLR)
+		;and	hl,de
 #else
       ld		a, (ix+[_decb]+rxbit)
       ld		b,a
       ioi	ld a,(DMCSR)
       and   a,b
 #endif
-      jr		nz,.cont1
 
-      ;FIXME - this should not be required!

```

Comment:
Can we remove the 5000-code that compares `rxbit` to `DMCSR`?  You've removed the relative jump over the `NBCR`-setting code and call to `dmaeth100_fire_rx`, so there's no point in making the comparison.  If uncertain, move the relative jump and `(NBCR)=1` code to inside the conditional (and get rid of the `_RAB6K` section entirely).